package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "gatsby-remark-mermaid",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Could you revert this version? This is part of the release process. It should not be part of a PR.

```suggestion
  "version": "2.1.0",
```